### PR TITLE
CRS-1790: Add offender last name to calculable sentence envelope.

### PR DESCRIPTION
**Pull request by justin-gardiner**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java [hunk 1]
```diff
@@ -869,6 +869,7 @@ private CalculableSentenceEnvelope determineCalculableSentenceEnvelope(OffenderB
         final var person = new uk.gov.justice.hmpps.prison.api.model.calculation.Person(
             offenderBooking.getOffender().getNomsId(),
             offenderBooking.getOffender().getBirthDate(),
+            offenderBooking.getOffender().getLastName(),
```
Comment (petergphillips) [inline src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java:872]: note that in Prison API we would normally try and capitalise as well e.g. https://github.com/ministryofjustice/prison-api/blob/e7942ead7a4316e7d150d644241540d011ba1f42/src/main/java/uk/gov/justice/hmpps/prison/service/MovementsService.java#L179

**Review by petergphillips — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java (hunk 1 above)
Comment (justin-gardiner) [reply src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java]: @petergphillips, ok I've updated

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**